### PR TITLE
[enriched/git] Add more git commit meta fields

### PR DESCRIPTION
**Pull request by zhquan**
This code extracts more authors meta info:
- Approved-by
- Co-authored-by
- Merged-by

For each one it will creates 10 new fields.
- suffixes: `_bots`, `_domains`, `_names`, `_org_names`, `_uuids`
- prefix: `non_authored_`

Example of Merged-by:
- merged_by_multi_bots
- merged_by_multi_domains
- merged_by_multi_names
- merged_by_multi_org_names
- merged_by_multi_uuids
- non_authored_merged_by_multi_bots
- non_authored_merged_by_multi_domains
- non_authored_merged_by_multi_names
- non_authored_merged_by_multi_org_names
- non_authored_merged_by_multi_uuids

Signed-off-by: Quan Zhou <quan@bitergia.com>

### CODE REVIEW

**Comment by jjmerchante** [comment]
LGTM

File: grimoire_elk/enriched/git.py [hunk 1]
```diff
@@ -86,8 +86,9 @@ class GitEnrich(Enrich):
     GIT_AOC_ENRICHED = "git_aoc-enriched"
 
     roles = ['Author', 'Commit']
-    meta_fields = ['acked_by_multi', 'co_developed_by_multi', 'reported_by_multi', 'reviewed_by_multi',
-                   'signed_off_by_multi', 'suggested_by_multi', 'tested_by_multi']
+    meta_fields = ['acked_by_multi', 'approved_by_multi', 'co_authored_by_multi', 'co_developed_by_multi',
```
Comment (sduenas) [inline grimoire_elk/enriched/git.py:89]: Perceval doesn't support this list of trailers. How is it possible the end in the enriched indices?

Comment (zhquan) [reply grimoire_elk/enriched/git.py:89]: Perceval fetches the entire commit message and elk parses the commit message to find these trailers and create them.

File: releases/unreleased/git-extracts-more-authors-meta-info.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    Example of Merged-by:
+    - merged_by_multi_bots
+    - merged_by_multi_domains
+    - merged_by_multi_names
+    - merged_by_multi_org_names
+    - merged_by_multi_uuids
+    - non_authored_merged_by_multi_bots
+    - non_authored_merged_by_multi_domains
+    - non_authored_merged_by_multi_names
+    - non_authored_merged_by_multi_org_names
```
Comment (sduenas) [inline releases/unreleased/git-extracts-more-authors-meta-info.yml:25]:
```suggestion
notes: >
    New information about the participants during the development and the
    reviewing processes are available in `Git` items. The Git backend extracts
    these data from `Approved-by`, `Co-authored-by`,  and `Merged-by`
    trailers that some commit messages might include. For each one of these
    trailers, new fields will be available with the suffixes: `_bots`, `_domains`,
    `_names`, `_org_names`, and `_uuids`. Also for each one of these suffixes,
    there will be extra fields with the prefix `non_authored_`.
    This is an example of the fields available for the the trailer `Merged-by`:
        - merged_by_multi_bots
        - merged_by_multi_domains
        - merged_by_multi_names
        - merged_by_multi_org_names
        - merged_by_multi_uuids
        - non_authored_merged_by_multi_bots
        - non_authored_merged_by_multi_domains
        - non_authored_merged_by_multi_names
        - non_authored_merged_by_multi_org_names
```

File: releases/unreleased/git-extracts-more-authors-meta-info.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+title: Git extracts more authors meta info
```
Comment (sduenas) [inline releases/unreleased/git-extracts-more-authors-meta-info.yml:2]:
```suggestion
title: Development and reviewing contributors fields for Git items
```

**Review by sduenas — CHANGES_REQUESTED** [review]
LGTM, would you  mind to update the changelog with my suggestion and rebase this PR?

**Review by sduenas — APPROVED** [review]
LGTM

**Pull request merged**